src/test/scala/ironoxide/FullIntegrationTest.scala

```diff
@@ -52,8 +53,8 @@ class FullIntegrationTest extends AsyncWordSpec with Matchers with EitherValues
 
   "DeviceCreateOpts" should {
     "create with DeviceName of null" in {
```

Comment:
The name of this test should change now to indicate that it's none, not null.

Reply:
Changed.